perceval/backends/core/github.py

```diff
@@ -68,7 +69,8 @@ class GitHub(Backend):
 
     :param owner: GitHub owner
     :param repository: GitHub repository from the owner
-    :param api_token: GitHub auth token to access the API
+    :param api_token: GitHub auth token(s) to access the API.
+        can be single token or comma separated list of tokens
```

Comment:
you could use the same description you wrote for the param `token` in `GitHubClient`